src/layouts/Sites.jsx

```diff
@@ -53,6 +56,20 @@ export default class Sites extends Component {
     const { siteNames } = this.state
     return (
       <>
+        {/* TODO: Move ThemeProvider to root of app during design system refactor */}
```

Comment:
we should add an issue and write the issue number within the comment - this allows devs to have more context on the issue and ensure that it's being tracked somewhere.

Reply:
Good point - let me create an issue

Reply:
Addressed in e831b47